_data/projects/php-asterisk-notation.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: PHP Asterisk Notation
+desc: >-
+  Asterisk notation for array access in PHP
+site: https://github.com/setnemo/asterisk-notation
+tags:
+- php
+- arrays
+- asterisk
+- php8
+- php74
+- asterisk-notation
+
+upforgrabs:
+  name: Up-For-Grabs
+  link: https://github.com/setnemo/asterisk-notation/labels/Up-For-Grabs
+stats:
+  issue-count: 0
```

Comment:
@setnemo This is autopopulated with our `Update Project Stats` PRs. Please remove it.